app/lib/request_helper.rb

```diff
@@ -0,0 +1,9 @@
+module RequestHelper
+  def self.get_header(header_name, request_headers)
+    request_headers[headerise(header_name)]
+  end
```

Comment:
Did you add this somewhere else in this codebase?

Reply:
I think I am using this in a few places in this PR now 🙂